package.json

```diff
@@ -1,20 +1,20 @@
 {
   "name": "magic-script-typings",
-  "version": "1.6.2",
+  "version": "1.7.0",
   "description": "Typescript definitions for MagicScript",
   "files": [
+    "egl",
     "gl",
     "jpeg",
     "lumin",
     "mxs",
     "png",
-    "uv",
-    "index.d.ts",
```

Comment:
do we want to remove this file from the repository if we're not using it?

Reply:
I thought it might be used by typedoc.  Let me verify it's not used by typedoc or vscode.

Reply:
Yep, just verified.  It's not used anywhere.  Removing...